tests/test_AnoGam.py

```diff
@@ -17,35 +17,46 @@ def test_name(self):
     def test_common_name(self):
         assert self.species.common_name == "Anopheles gambiae"
 
-    # QC Tests. These tests are performed by another contributor
-    # independently referring to the citations provided in the
-    # species definition, filling in the appropriate values
-    # and deleting the pytest "skip" annotations.
-    @pytest.mark.skip("Population size QC not done yet")
     def test_qc_population_size(self):
-        assert self.species.population_size == -1
+        # based on theta = 4 Ne u since diversity in most pops is 1.5%
+        assert pytest.approx(self.species.population_size, 0.015 / 3.5e-9)
 
-    @pytest.mark.skip("Generation time QC not done yet")
     def test_qc_generation_time(self):
-        assert self.species.generation_time == -1
+        assert self.species.generation_time == 1 / 11
 
 
 class TestGenomeData(test_species.GenomeTestBase):
 
     genome = stdpopsim.get_species("AnoGam").genome
+    mut_rate = 3.5e-9
```

Comment:
The paper provided lists three mutation rates from Drosophila: 2.5e-9, 3.5e-9, and 5.5e-9. The number in the code is 5.49e-9, which doesn't appear in the Ag1000g paper, but I assume is a more precise number from Schrider et al, the source of the third number. Based on only the Ag1000g paper I would have chosen 3.5e-9, since (a) it's the one in the middle, and (b) it's the number which is used to scale some of the plots. But, maybe the third one is a better estimate? If so, we should say something about why we're choosing that one. And, add the citation to the original Drosophila paper that we get the estimate from.

Reply:
yes 5.49e-9 is what we used for the demographic analysis in the paper I believe. @dschride does that sound right to you?

Reply:
The main demographic analysis gives ranges of times, corresponding to the range of mutation rates. I didn't find any demograhpic inference that used only 5.5e-9. But, maybe I missed it? Is that the mutation rate for  the times for the AnoGam demographic model you've put in?

Reply:
yep. IIRC that's the rate for all of the Stairwayplot runs we did (1 for each population)

Reply:
crap this is wrong! we used 3.5e-9 
![image](https://user-images.githubusercontent.com/7674197/114630848-469e9400-9c70-11eb-8c82-c969aceaf32a.png)

so that should be edited on the QC PR